userguide/payment/includes/payment-states.adoc

```diff
@@ -8,7 +8,7 @@ The user can then submit additional requests for the same payment (capture, refu
 The payments have a state which will drive the set of possible subsequent operations available. For instance, it is possible to make a capture call against a payment in a state `AUTH_SUCCESS` but it is not possible to make such an operation against a payment in an `AUTH_ERRORED` state.
 The set of payment state transitions are configured in an https://github.com/killbill/killbill/blob/master/payment/src/main/resources/org/killbill/billing/payment/PaymentStates.xml[xml file] and the Kill Bill payment subsystem is in charge to enforce the transitions. The following diagram below shows all the possible transitions:
 
-graphviz::assets/payment_states.dot[format=svg,align=center]
```

Comment:
This is actually correct, but Qodana isn't smart enough to know it.

Reply:
👍 Maybe the graphviz settings need to be tweaked...

Reply:
Yes, I'll look into it.